dicom_standard/extract_attributes.py

```diff
@@ -24,13 +28,13 @@ def attribute_table_to_list(table_div):
 
 
 def attribute_table_to_json(table):
-    attribute_dict = {}
+    attributes = []
     for attr in table:
-        attr_slug = pl.create_slug(attr['tag'])
-        attribute_dict[attr_slug] = attr
-        attribute_dict[attr_slug]['tag'] = attr['tag'].upper()
-        attribute_dict[attr_slug]['retired'] = True if attr['retired'] == 'RET' else False
-    return attribute_dict
+        attr['id'] = pl.create_slug(attr['tag'])
```

Comment:
The rationale for updating the JSON structure is detailed here: https://github.com/innolitics/dicom-standard/issues/24#issuecomment-591579455

The goal was to have the JSON files act as database tables.

Reply:
Okay, gotcha. Thanks for linking the issue here again, that makes a lot more sense now-- I had missed it in your PR description.

It may make sense for us to assert that ID's are unique when assembling the table-- it may make the time complexity of this loop jump to O(N^2) but I don't think that's a big deal since this code only has to be run once. The consistency check would just provide an extra guarantee that all our assumptions are being maintained.